bebop_driver/src/bebop_driver_nodelet.cpp

```diff
@@ -59,7 +59,6 @@ int BebopPrintToROSLogCB(eARSAL_PRINT_LEVEL level, const char *tag, const char *
 }  // namespace util
 
 BebopDriverNodelet::BebopDriverNodelet()
-  : bebop_ptr_(new bebop_driver::Bebop(util::BebopPrintToROSLogCB))
```

Comment:
This is a bit dangerous, since now there is no guarantee that this pointer is not NULL and the object is initialized when for example a callback is called when `onInit()` method is not finished. I am not sure if that is actually a case when using nodelets, however I like the safety that this early allocation provides. 

I suggest that we leave the original constructor untouched and pass/save `bebop_ip` when calling `Connect()`.